src/network/networkctl.c

```diff
@@ -2728,6 +2844,8 @@ static int help(void) {
                "  -s --stats             Show detailed link statics\n"
                "  -l --full              Do not ellipsize output\n"
                "  -n --lines=INTEGER     Number of journal entries to show\n"
+               "     --json=pretty|short|off\n"
+               "                         Generate JSON output\n"
```

Comment:
`--json=...` affects `networkctl status`, but not `networkctl list`. Do you plan to add that too?